distro/pom.xml

```diff
@@ -183,6 +183,20 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.openmrs.maven.plugins</groupId>
+        <artifactId>openmrs-packager-maven-plugin</artifactId>
+        <version>1.5.0-SNAPSHOT</version>
```

Comment:
We need to do a release of the Maven plugin before this can be incorporated. I think it's OK to tolerate SNAPSHOTs in the properties that are driving the distro configuration, but completely wrong to do that for Maven plugins.

Reply:
Yes for sure. That's precisely the plan. I'd simply done this in awaiting a release of the plugin.

Reply:
Done. The Plugin used is 1.7.0.